scripts/pnodespy.py

```diff
@@ -84,13 +84,13 @@ def Get_Location(node: Iir) -> LocationType:
             @export
             @BindToLibGHDL("{classname}__get_{kname_lower}")
             def Get_{kname}(obj: Iir) -> {rtype}:
-                \"\"\"{gettercomment}\"\"\"
```

Comment:
Should we really remove this or can @tgingold provide this from Ada files?

Reply:
I think that most of the getters/setters don't have any description/comment in Ada. Hence, we might just write a general comment here.